src/cmps/multi-step-form.jsx

```diff
@@ -3,40 +3,52 @@ import { addUser } from "../store/user.actions";
 import { Form, Formik } from "formik";
 import FormStepsWrapper from "./form-steps-wrapper";
 import { SiteContext } from "../store/context";
+import * as Yup from 'yup';
+
+const validationSchema = Yup.object().shape({
+	first_name: Yup.string().required('Required'),
+	email: Yup.string().email('Invalid email').required('Required'),
+});
+
 
 export default function MultiStepForm() {
-    const {
-        data
-    } = useContext(SiteContext)
-
-
-    // const makeRequest = (formData) => {
-    //     dispatch(addUser({
-    //         first_name: formData.first_name,
-    //         last_name: formData.last_name,
-    //         email: formData.email,
-    //         birthDate: formData.birthDate,
-    //     }))
-    //     console.log("Form Submitted", formData);
-    // }
-
-    const handleSubmit = () => {
-
-    }
-
-    return (
-        <div className='form'>
-            <Formik
-                // validationSchema={stepOneValidationSchema}
-                initialValues={data}
-                onSubmit={handleSubmit}
-            >
-                {(values) =>
-                    <Form>
-                        <FormStepsWrapper values={values} />
-                    </Form>
-                }
-            </Formik>
-        </div>
-    )
+	const {
+		data
+	} = useContext(SiteContext)
+
+
+	// const makeRequest = (formData) => {
+	//     dispatch(addUser({
+	//         first_name: formData.first_name,
+	//         last_name: formData.last_name,
+	//         email: formData.email,
+	//         birthDate: formData.birthDate,
+	//     }))
+	//     console.log("Form Submitted", formData);
+	// }
+
+	const handleSubmit = () => {
+
+	}
+
+	return (
+		<div className='form'>
+			<Formik
+				// validationSchema={stepOneValidationSchema}
+				initialValues={data}
+				onSubmit={handleSubmit}
+				validationSchema={validationSchema}
+			>
+				{(formikProps) => {
```

Comment:
You can also destructure it like so:
`{({values}) => {.....`

It'll prevent a lot of nesting (:

Reply:
Thanks for review,
The purpose was to pass this props to child components, some of them are using the values and some of them are using other properties/methods that Formik exposes through it child params.
I think we need to schedule a meeting for the next steps in order to decide approach for the architecture that we would be using.